go.mod

```diff
@@ -133,10 +133,5 @@ require (
 )
 
 // When updating replace rules, make sure to also update the rules in integration/client/go.mod and api/go.mod
-replace (
-	github.com/gogo/googleapis => github.com/gogo/googleapis v1.3.2
-
-	// urfave/cli must be <= v1.22.1 due to a regression: https://github.com/urfave/cli/issues/1092
-	github.com/urfave/cli => github.com/urfave/cli v1.22.1
-	google.golang.org/genproto => google.golang.org/genproto v0.0.0-20200224152610-e50cd9704f63
```

Comment:
The github.com/gogo/googleapis and google.golang.org/genproto replace rules had to be removed, as module resolution failed with them (golang.org/x/oauth2 and other failures)